apps/rellotge/rellotge.js

```diff
@@ -0,0 +1,131 @@
+// drawWidgets() is added by bootloader.js when loading a clock app, but when you upload via the IDE it just 
+// resets the watch and skips out running bootloader.js completely. So add the relevant code from the bootloader. 
+var WIDGETPOS = {
+  tl: 32,
+  tr: g.getWidth() - 32,
+  bl: 32,
+  br: g.getWidth() - 32
+};
+var WIDGETS = {};
+
+function drawWidgets() {
+  for (var w of WIDGETS) w.draw();
+}
+
+require("Storage").list().filter(a => a[0] == '=').forEach(
+  widget => eval(require("Storage").read(widget)));
+setTimeout(drawWidgets, 100);
+
+// Example application code
+// Taken from https://github.com/espruino/BangleApps/blob/master/apps/sclock/clock-simple.js
+(function() {
+
+  const timeFontSize = 1;
+  const dateFontSize = 2;
+  const gmtFontSize = 10;
+  const font = "12x20";
+
+  const xyCenter = g.getWidth() /9;
+  const yposTime = 55;
+  const yposDate = 130;
+  const yposYear = 175;
+  const yposGMT = 220;
+  const leshores = ["Les dotze","La una","Les dues","les tres","Les quatre","Les cinc","Les sis","Les set","Les vuit","Les nou","Les deu","Les onze","Les dotze","La una","Les dues","Les tres","Les quatre","Les cinc","Les sis","Les set","Les vuit","Les nou","Les deu","Les onze","Les dotze"];
+  const leshores2 = ["d\'una\r\nel mati","de dues\r\ndel mati","de tres\r\ndel mati","de quatre\r\ndel mati","de cinc\r\ndel mati","de sis\r\ndel mati","de set\r\ndel mati","de vuit\r\ndel mati","de nou\r\ndel mati","de deu\r\ndel mati","d'onze\r\ndel mati","de dotze\r\ndel mati","d'una\r\nde la tarda","de dues\r\nde la tarda","de tres\r\nde la tarda","de quatre\r\nde la tarda","de cinc\r\nde la tarda","de sis\r\nde la tarda","de set\r\nde la tarda","de vuit\r\nde la tarda","de nou\r\ndel vespre","de deu\r\ndel vespre","d'onze\r\ndel vespre","de dotze"];
+
+  function drawSimpleClock() {
+    g.clear();
```

Comment:
With widgets, it's better here to do this to avoid overwriting them:

```suggestion
    g.clearRect(Bangle.appRect);
```

(and then we do one initial clear at [the start like you have here](https://github.com/espruino/BangleApps/blob/d1ab9e2ebf949ee29b78a4e00904417af7c1845b/apps/rellotge/rellotge.js#L122-L123))